### PR TITLE
Added installation of matlab functions

### PR DESCRIPTION
**Pull request by francesco-romano**
Still in the documentation it should be stated that the following folders should be added to the matlab path (either every time in a script, or just once by editing the `pathdef.m` file)

```
${CMAKE_INSTALL_PREFIX}/mex
${CMAKE_INSTALL_PREFIX}/mex/mexwbi-wrappers
${CMAKE_INSTALL_PREFIX}/mex/mexwbi-utilities 
```

cc @DanielePucci @traversaro @naveenoid 

This will fix https://github.com/robotology/mex-wholebodymodel/issues/20

### CODE REVIEW

**Comment by traversaro** [comment]
cc @francesco-romano  The tests are passing?

**Comment by traversaro** [comment]
@naveenoid any opinions ?

**Comment by francesco-romano** [comment]
Tests are passing

File: CMakeLists.txt [hunk 1]
```diff
@@ -4,8 +4,7 @@
 
 cmake_minimum_required(VERSION 2.8)
 
-SET(PROJECTNAME wholeBodyModel)
-PROJECT(${PROJECTNAME})
+PROJECT(wholeBodyModel)
```
Comment (naveenoid) [inline CMakeLists.txt]: do we want to call the project wholeBodyModel and not mexWholeBodyModel? Wont this be confusing?

**Comment by naveenoid** [comment]
Tests are passing. Seems fine. Im good to merge. You can do it after discussing my line note https://github.com/robotology/mex-wholebodymodel/pull/53/files#r43816744

File: CMakeLists.txt (hunk 1 above)
Comment (traversaro) [inline CMakeLists.txt]: The `PROJECT_NAME` variable is used in the cmake to name the mex file. If we want to call it `mexWholeBodyModel`, we will have to change also all the matlab files.

Comment (naveenoid) [inline CMakeLists.txt]: Ah yes I understand. Then Im cool with the PR.


On Tue, Nov 3, 2015 at 11:55 PM, Silvio Traversaro <notifications@github.com

Comment (francesco-romano) [inline CMakeLists.txt]: And anyway I did not change anything.
That is the old name.

**Pull request merged**